airbyte-integrations/connectors/source-railz/unit_tests/pyproject.toml

```diff
@@ -29,3 +30,8 @@ pytest-mock = "^3.6.1"
 requests-mock = "^1.9.3"
 pytest = "^6.2"
```

Comment:
You don't need it here anymore, I would remove it. Or update to pytest 6. I think you can remove the whole section for dev dependencies.